gnat2goto/COMPILING.md

```diff
@@ -0,0 +1,45 @@
+# Build Dependencies
+
+**To build GnatColl GPL 2016 and gnat2goto you must use a gnat GPL 2016 compiler and gprbuild (see the GnatColl user documentation)**
+
+We need the 2016 versions of the GNAT Ada compiler, the gprbuild system and the
+GNATColl utility library to build GNAT2GOTO.
+
+Note: you will *not* be able to build using the debian/ubuntu gnat and gprbuild.
```

Comment:
Is it worth expanding that note a little bit - "gnat2goto needs the specific versions of GNAT listed in this file. Debian/Ubuntu packaged builds of gnat and gprbuild do not match this version."

Reply:
I'm actually not sure how specific it needs to be. It doesn't work with the Ubuntu version, but the it works fine with "the" version presented on AdaCore as the 2016 version.